posthog/tasks/usage_report.py

```diff
@@ -326,10 +326,20 @@ def get_teams_with_event_count_lifetime() -> List[Tuple[int, int]]:
 
 
 @timed_log()
-def get_teams_with_billable_event_count_in_period(begin: datetime, end: datetime) -> List[Tuple[int, int]]:
+def get_teams_with_billable_event_count_in_period(
+    begin: datetime, end: datetime, count_distinct: bool = False
+) -> List[Tuple[int, int]]:
+    # count only unique events
```

Comment:
could you add a Harry style comment here, sth like:
"Duplicate events will be eventually removed by ClickHouse and likely came from our library or pipeline. We shouldn't bill for these. However counting unique events is much more expensive, so the query is likely to OOM if ran over more than a day."